pages/docs/concepts/asyncapi-document/variable-url.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
Why `variable-url.md`? I think it makes more sense to have `url-variables.md`. Can you please update accordingly?